tests/components/modbus/test_modbus_sensor.py

```diff
@@ -26,6 +26,7 @@
     DATA_TYPE_STRING,
     DATA_TYPE_UINT,
 )
+from homeassistant.components.modbus.sensor import create_data_structure
```

Comment:
We try to make all tests as black box testing, that is the test does not know the implementation detail.

Reply:
Make sense.